Menu/Menu.js

```diff
@@ -1,10 +1,17 @@
-
-const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
-
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector('.menu-button');
 // Using your menuButton reference, add a click handler that calls toggleMenu
+
+
+menuButton.addEventListener('click', function() {
+  // Toggle the "menu--open" class on your menu refence. 
+  //console.log('menu');
+  menu.classList.toggle('menu--open');
```

Comment:
this works and is fine, but the assignment wanted you to create a function (toggleMenu) that toggled the classList on menu and then put that function in as the call back `menuButton.addEventListener('click', toggleMenu)` but this is no different, you just wrote the function inside instead